script2.js

```diff
@@ -0,0 +1,125 @@
+let noOfPlayers, noOfDice;
+let diceRolls = [];
+let optimumNumbers = [];
+let mode = "playerCount";
+changeInstructions(mode);
```

Comment:
Similar to your earlier version, rather than calling this function here, you should have set the default look you needed in the html